beam_conversion/feko.py

```diff
@@ -28,17 +28,21 @@ def load(self):
         f_ground = {}  # store a map for freq → f_ground
         reading_power_radiated = False  # toggle whether we look for radiated power
         farfield = ""
+        freq_list = []
         print ("Loading frequencies: ", end = "")
         for line in data:
             if skip:
+                ## ignoring radiated power for a while
                 # if we find a header with radiated power, start looking
-                if "The directivity/gain is based on an active power of" in line:
-                    reading_power_radiated = True
-                    radiated_power = float(line.split()[-2]) #normalization
```

Comment:
Since radiated_power and f_ground are being removed, the read_power_radiated stuff seems like it should be removed near [#L62.](https://github.com/lusee-night/luseepy/pull/10/files#diff-efc2c54a22fd6838e3db4db56ae820390e3f337a143e0c534ff0251acdb21b45R62)